code/style.css

```diff
@@ -1,7 +1,261 @@
+* {
+  /* Universal selector: removes default stylings of all html tags */
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: whitesmoke;
+}
+
+/* HEADER */
+header {
+  background-color: black; /* This is a back up for the background-image, in case doesn't load or if some browsers don't support the image, this color will show */
+  background-image: url(./Images/Gymnastics-picture.jpg.crdownload);
+  height: 250px;
+  width: 100%;
+  background-position: center; /* Center the image */
+  background-repeat: no-repeat; /* Do not repeat the image */
+  background-size: contain;
+  border: 2px solid;
+}
+
+div.header {
+  display: flex;
+}
+
+.logo-text {
+justify-content: center;
+font-size: 1.3rem;
+margin-top: 15px;
+color: rgb(235, 100, 100);
+}
+
+span {
+  color: rgb(238, 185, 194);
+  font-weight: bold;
+}
+u {
+  color:blue;
+}
+
+.logo-img {
+  width: 50px;
+  height: 50px;
+  margin: 8px;
+  border-radius: 50%;
+  border: 1px solid rgb(235, 100, 100);
+}
+
+.nav-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: black; /* For browsers that does not support gradients*/
+
+}
+
+a {
+  padding: 1rem;
+  font-size: 1.1rem;
+  cursor: pointer;
+  color: rgb(238, 185, 194);
+  text-decoration: none;
+  font-weight: bold;
+}
+
+a:hover {
+  transform: scale(1.2, 1.2);
+}
+
+/* FORM SECTION */
+
+.form-section {
+  margin:20px 10px;
+  padding:40px;
+  background-color: rgb(60, 141, 173);
+  border-radius: 20% 10%;
+}
+
+.register-heading {
+  text-align: center;
+  font-family: 'Merriweather', serif;
+  margin-bottom: 25px;
+  color: rgb(252, 240, 200);
+  text-transform: uppercase;
+}
+
+label {  
+  color: rgb(252, 240, 200);
+  font-weight: bolder;
+
+}
+
+.labels {
+  display:block;
+  padding-left: 10px;
+  margin: 10px 0px;
+}
+
+.inputs {
+  display: block;
+  width: 100%;
+  height: 40px;
+  border:none;
+  margin-bottom: 20px;
+}
+/* Pseudo-class selector: select elements based on a certain state */
+.inputs:focus { 
+   outline: none; /*removes default outline of all inputs */
+}
+/* Pseudo-element selector: select and style a part of an element*/
+input::placeholder { 
+  padding-left: 15px;
+  opacity:0.5;
+}
+/* Attribute selectors: select elements based on an attribute or attribute value */
+input[type="checkbox"] {
+  cursor: pointer;
+}
 
-.active + .answer {
-  /* some styling here */
+.button {
+  display: block;
+  margin:20px auto;
+  width: 90%;
+  height: 50px;
+  background-color: rgb(18, 93, 152);
+  border:none;
+  border-radius: 20px;
+  color: rgb(252, 240, 200);
+  font-size: 24px;
+  cursor: pointer;
+}
+
+.button:hover {
+  transform: scale(1.1, 1.1);
+}
+
+/* ACCORDION SECTION */
+
+.accordion-section {
+  margin:20px 10px;
+  padding:30px;
+  background-color: rgb(145, 31, 39);
+  border-radius: 10% 20%;
```

Comment:
Cool that you experimented with different % on the two sections (accordion and the form section). That made a very nice effect for the viewer